launcher/ios/main.m

```diff
@@ -0,0 +1,28 @@
+/*
+ * main.m, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#import <UIKit/UIKit.h>
+
+void launchGame(int argc, char * argv[]) {
+	@autoreleasepool {
+		__auto_type app = UIApplication.sharedApplication;
+		__auto_type qtNativeWindowIndex = [app.windows indexOfObjectPassingTest:^BOOL(__kindof UIWindow * _Nonnull window, NSUInteger idx, BOOL * _Nonnull stop) {
+			return [NSStringFromClass([window class]) isEqualToString:@"QUIWindow"];
+		}];
+		NSCAssert(qtNativeWindowIndex != NSNotFound, @"Qt native window doesn't exist");
+
+		__auto_type qtNativeWindow = app.windows[qtNativeWindowIndex];
+		qtNativeWindow.hidden = YES;
+		[qtNativeWindow.rootViewController.view removeFromSuperview];
+		qtNativeWindow.rootViewController = nil;
+		if (@available(iOS 13.0, *))
```

Comment:
iOS 13.0 is minimal compatible version? Does it make sense to have a parameters?

Reply:
asks someone who doesn't know development for iOS ;)

minimum supported iOS is 10.0. This is a runtime check for current iOS version to use a symbol that is available only starting from iOS 13, attempt to access it on iOS < 13 will result in a crash.